apps/andi/lib/andi_web/live/ingestion_live_view/transformations/transformation_form.ex

```diff
@@ -19,36 +19,46 @@ defmodule AndiWeb.IngestionLiveView.Transformations.TransformationForm do
 
     {:ok,
      assign(socket,
-       transformation_changeset: transformation_changeset
+       transformation_changeset: transformation_changeset,
+       visibility: "collapsed"
      )}
   end
 
   def render(assigns) do
+    action =
+      case assigns.visibility do
+        "collapsed" -> "EDIT"
+        "expanded" -> "MINIMIZE"
+      end
+
     ~L"""
 
-    <%= f = form_for @transformation_changeset, "#", [ as: :form_data, phx_change: :validate, class: "transformation-item"] %>
-        <div class="transformation-header">
-          <h3 class="transformation-header-name"> <%= transformation_name(f) %> </h3>
-          <div class="transformation-edit-buttons">
-            <span class="material-icons move-button move-up move-up-<%= @transformation_changeset.changes.id %>" phx-click="move-transformation" phx-value-id=<%= @transformation_changeset.changes.id %> phx-value-move-index="-1">arrow_upward</span>
-            <span class="material-icons move-button move-down move-down-<%= @transformation_changeset.changes.id %>" phx-click="move-transformation" phx-value-id=<%= @transformation_changeset.changes.id %> phx-value-move-index="1">arrow_downward</span>
-          </div>
-        </div>
-        <%= hidden_input(f, :id, value: @transformation_changeset.changes.id) %>
-        <div class="transformation-form">
-          <div class="transformation-form__name">
-            <%= label(f, :name, "Name", class: "label label--required") %>
-            <%= text_input(f, :name, class: "transformation-name input transformation-form-fields", phx_debounce: "1000") %>
-            <%= ErrorHelpers.error_tag(f.source, :name, bind_to_input: false) %>
+    <div class="transformation-form-container">
```

Comment:
Checked this out to test locally and looks great, nice work.

Only note is that this div seems unnecessary. 

The `bottom-margin` applied to this new `transformation-form-container` can instead be applied to `transformation-item`. `transformation-item` already exists as a selector for styles that apply to the entirety of the transformation "item" or "card" (header + form as one unit)

Reply:
Done.